controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -197,6 +245,7 @@ func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&ocsv1.OCSInitialization{}).
 		Owns(&corev1.Service{}).
 		Owns(&corev1.Secret{}).
+		Owns(&promv1.Prometheus{}).
```

Comment:
What about the other unique resources you are reconciling? 
I can detect a ConfigMap (PrometheusKubeRBACConfigMap), a Service (PrometheusService), an Alertmanager and a ServiceMonitor

Reply:
Updated to own the missing resources